lib/index.js

```diff
@@ -80,6 +86,16 @@ Optimizely.prototype.track = function(track) {
   var revenue = track.revenue();
   if (revenue) payload.revenue = revenue *= 100;
   push(['trackEvent', track.event(), payload]);
+
+  var optimizelyClientInstance = window.optimizelyClientInstance;
+  if (optimizelyClientInstance && optimizelyClientInstance.track) {
+    var attributes = track.properties();
+    var optimizelyOptions = track.options('Optimizely');
+    var userId = track.userId() || optimizelyOptions.userId;
```

Comment:
@mikeng13 One last thing -- I actually think it's better to have the `optimizelyOptions` precede the Segment userId. If a developer is explicitly going out of their way to set an Optimizely userId, we should respect that over the Segment userId.

```
var userId = optimizelyOptions.userId || track.userId();
```